spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/NamedConfigMapContextToSourceDataProvider.java

```diff
@@ -137,8 +135,4 @@ public KubernetesClientContextToSourceData get() {
 
 	}
 
-	private Supplier<String> appName(Environment environment, NormalizedSource normalizedSource) {
```

Comment:
when computing sources for secrets, I added this exact same logic there, in `commons` dependency. I decided to do the same thing here, remove this logic from here and add it to `commons`. Basically this just moves things around and makes the implementations on par (secrets and configmaps).